addons/rating/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,25 @@
+class Extended_Init_EventHandlers {
+    class CAManBase {
+        class ADDON {
+            serverInit = QUOTE(_this call FUNC(initMan));
```

Comment:
It may be a personal preference thing, but I don't think any config based XEHs should be defined if there is a case where it should never run (especially when the default case is that it shouldn't run).

My proposed solution:
On the server postInit you wait until the settings are finished initializing, only where there is a use case (one or both GVARs are false) then you install the class based event handler with the retroactive flag.

The current solution:
Even if there's no use case for it (both GVARs are true) the event handler gets installed, and will run (and exit) for every unit spawned.

Just seems needlessly wasteful.